cmds/ps/ps_linux.go

```diff
@@ -231,8 +232,7 @@ func (pT *ProcessTable) LoadTable() error {
 		if pf.Match([]byte(fi.Name())) {
 			p := &Process{}
 			if err := p.Parse(fi.Name()); err != nil {
-				log.Print(err)
-				return err
+				parsingError = err
 			}
 			pT.table = append(pT.table, *p)
```

Comment:
HMMM! Deep catch! I'll change that, is really unnecessary. Review my next commit for this. Hold the PR